src/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -174,7 +181,7 @@ module Stat = struct
     let sectors_to_bytes = Int64.mul hw_sector_size in
     let read_bytes = sectors_to_bytes (List.nth stats 2) in
     let write_bytes = sectors_to_bytes (List.nth stats 6) in
-    let res = stats @ inflight_stats @ [read_bytes; write_bytes] in
+    let res = (Xapi_stdext_std.Listext.List.take 11 stats) @ inflight_stats @ [read_bytes; write_bytes] in
```

Comment:
So this takes the first 11 stats and ignores the remaining, newer ones. I think this is an opportunity for micro optimization to avoid `@` (which is expensive):

```
let res = List.concat 
  [ Xapi_stdext_std.Listext.List.take 11 stats
  ; inflight_stats
  ; [read_bytes; write_bytes] 
  ]
```